index.html

```diff
@@ -0,0 +1,191 @@
+        <li><a class="nav-link" href="#while_statements">while statements</a></li>
+        <li><a class="nav-link" href="#Function_declaration">Function declaration</a></li>
+        <li><a class="nav-link" href="#Reference">Reference</a></li>
+       
+    </ul>
+</nav>
+
+<main id="main-doc">
+  <section class="main-section" id="Introduction"> 
+    <header>
+      Introduction
+    </header>
+    <p>JavaScript is a cross-platform, object-oriented scripting language. It is a small and lightweight language. Inside a host environment (for example, a web browser), JavaScript can be connected 
+        to the objects of its environment to provide programmatic control over them.</p>
+  </section>
+  <section class="main-section"id="What_you_should_know_already" >
+    <header>
+      What you should know already
+    </header>
+    <p>This guide assumes you have the following basic background:</p>
+    <ul>
+        <li>General understanding of the Internet and the World Wide Web (WWW).</li>
+        <li>Good working knowledge of HyperText Markup Language (HTML).</li>
+    </ul>
+  </section>
+  <section class="main-section" id="Javascript_and_Java"> 
+    <header>
+      JavaScript and Java
+    </header>
+    <p>JavaScript and Java are similar in some ways but fundamentally different in some others. </p>
+
+    <p>In contrast to Java's compile-time system of classes built by declarations, JavaScript 
+        supports a runtime system based on a small number of data types representing numeric, Boolean, and string values. 
+        JavaScript has a prototype-based object model instead of the more common class-based object model. The prototype-based model provides dynamic inheritance; that is, what is inherited can vary for individual objects. JavaScript also supports functions without any special declarative requirements. 
+        Functions can be properties of objects, executing as loosely typed methods.</p>
+    <ul>
+        <li>They are similar in some ways</li>
+        <li>They are fundamentally different in some ways</li>
+    </ul>
+  </section>
+  <section class="main-section" id="Hello_world"> 
+    <header>
+      Hello world
+    </header>
+    <code>function greetMe(yourName) { alert("Hello " + yourName); }
+        greetMe("World");</code>
+  </section>
+  <section class="main-section" id="Variables">
+    <header>
+      Variables
+    </header>
+    <p>You use variables as symbolic names for values in your application. The names of variables, 
+        called identifiers, conform to certain rules.</p>
+    </section>
+  <section class="main-section" id="Declaring_variables"> 
+    <header>
+      Declaring variables
+    </header>
+    <p>You can declare a variable in three ways:
+    </p>
+    <code>var x = 42.</code>
+    <code>x = 42.</code>
+    <code>let y= 13.</code>
+  </section>
+   <section class="main-section" id="Variables_scope"> 
+    <header>
+      Variables scope
+    </header>
```

Comment:
✔️ Good job done using Ids. But it seems your code has so many ids. 🤔 I think just one or two is okay for a page. Many ids can create problems for managing styles in one place
id and header issues

✔️ Good semantics job by using header. But some headers seems to have no style class. 🤔 I think If you have others pages where headers change it will be difficult to manage. Kindly work on it.
id and header issues2